src/platform/forms-system/src/js/state/helpers.js

```diff
@@ -478,14 +478,21 @@ export function updateItemsSchema(schema, fieldData = null) {
 
 /**
  * This is the main sequence of updates that happens when data is changed
- * on a form. Most updates are applied to the schema, except that the data
- * is updated to remove newly hidden data
+ * on a form. Most updates are applied to the schema. And by default the data
```

Comment:
All the changes in this file are also part of this PR: https://github.com/department-of-veterans-affairs/vets-website/pull/11589